squidb/src/com/yahoo/squidb/sql/VirtualTable.java

```diff
@@ -72,8 +74,11 @@ public String toString() {
      */
     @Override
     public void appendCreateTableSql(StringBuilder sql, PropertyVisitor<Void, StringBuilder> propertyVisitor) {
-        sql.append("CREATE VIRTUAL TABLE IF NOT EXISTS ").append(getExpression()).append(" USING ").append(moduleName)
-                .append('(');
+        sql.append("CREATE VIRTUAL TABLE ");
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
Good catch, I had noticed this yesterday as well. We'll need to revisit this if we ever decide to allow bundling other versions of SQLite, but for now this is good.